front/src/utils/__tests__/getColorForShowIds.test.ts

```diff
@@ -0,0 +1,21 @@
+import { getUniqueColorsForShowIds } from '../getColorForShowId';
+
+//returns true if array values are unique otherwise returns false;
+const isUnique = (arr: string[]): boolean => new Set(arr).size === arr.length;
+
+describe('getUniqueColorsForShowIds', () => {
+  it('returns 2 unique colors', () => {
+    const colors = getUniqueColorsForShowIds([1000, 1001]);
+    expect(isUnique(colors)).toBeTruthy();
+  });
+
+  it('returns 4 unique colors', () => {
+    const colors = getUniqueColorsForShowIds([1000, 2000, 3000, 4000]);
```

Comment:
- Can we change the IDs to be more unique, so like random numbers without zeros?
- Do you mind adding a test to test for inputting 1 ID and expecting an array with length of 1?

Reply:
Ok. requested changes should be in